### ISSUE TITLE
I can't log in with a Shibboleth server 

### ISSUE THREAD

**Issue by lmaestro** [owncloud/client#2780]
### Expected behavior
I can log in with a Shibboleth server

### Actual behavior
I am not able to log in with a shibboleth server

### Steps to reproduce
1. Launch the app
2. Introduce the shibboleth server and the credentials 
3. The credentials windows is opened again 

### Client configuration
Version 1.8.0-nightly20150202(build 1963)

Operating system: OS X 10.10.1 and Windows 8.1

###Logs
https://gist.github.com/lmaestro/1ee75696bf3da076a139

**Comment by dragotin** [comment]
This is probably because the new discovery code has some hickup with Shib yet. @guruz

**Comment by ogoffart** [comment]
The problem was in the wizard:
(From lucia log:
```
OCC::EntityExistsJob created for  QUrl( "https://test1.owncloud.com/oc7-shib" )  querying "/"
void OCC::ShibbolethCredentials::slotReplyFinished(QNetworkReply *) detected redirect, will open Login Window
```

It was querying / which was a redirect to app/files,  which shiboleth would understand as meaning the authentication is needed

**Comment by ogoffart** [comment]
And it was a duplicate of #2739

**Comment by lmaestro** [comment]
It is fixed in the latest version version 1.8.0-nightly20150205 (build 1984)
